fab/fab/operations/supervisor.py

```diff
@@ -147,19 +147,15 @@ def show_periodic_server_whitelist_message_and_abort(env):
     )
 
 
-def set_pillowtop_supervisorconf():
-    # Don't run if there are no hosts for the 'django_pillowtop' role.
-    # If there are no matching roles, it's still run once
-    # on the 'deploy' machine, db!
-    # So you need to explicitly test to see if all_hosts is empty.
-    if env.all_hosts and _check_in_roles(ROLES_PILLOWTOP):
-        _rebuild_supervisor_conf_file(
-            'make_supervisor_pillowtop_conf',
-            'supervisor_pillowtop.conf',
-            {'pillow_env_configs': get_pillow_env_config()}
-        )
+def copy_pillowtop_supervisorconf():
+    services_dir = "/services/supervisor/"
```

Comment:
nit: does this make for a double slash when combined with the lines below?

Reply:
It occurs to me that we may simply want to host these config files in `/home/cchq/www/<env>/services` or something like that, since the coupling with the code release doesn't really make sense anymore. Do you agree, and do you want to do that now (as you move these to ansible) or in a later refactor?

Reply:
Totally agree. I was a little hesitant to add that to this PR as that also means we need to change the supervisor configuration to look in a couple of places. Wanted to keep the parts moving around here small since we've had some issues deploying recently